StreamChatIntegrationSource/AppDelegate.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  AppDelegate.swift
+//  StreamChatIntegration-SPM
+//
+//  Created by Boris Bielik on 16/11/2021.
+//
+
+import UIKit
```

Comment:
I'm guessing that this file is used by all the integration projects, right?

Reply:
That's correct. All integrations do shared 1 source code.